extlibs/BUILD.sfml

```diff
@@ -1,21 +1,30 @@
-load("@rules_cc//cc:defs.bzl", "cc_library")
```

Comment:
Did you mean to remove this? Using the built-in cc_library is deprecated afaik.

Reply:
Sorry, my mistake. Thanks for the review!